### PR TITLE
Set 1d plot layer based on priority.

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

File: vcs/vcsvtk/pipeline1d.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def plot(self, data1, data2, tmpl, grid, transform, **kargs):
                 self._context().canvas.plot(m, donotstoredisplay=True)
 
         ren2 = self._context().createRenderer()
-        ren2.SetLayer(1)
+        self._context().setLayer(ren2, tmpl.data.priority)
```
Comment (doutriaux1) [inline vcs/vcsvtk/pipeline1d.py]: @danlipsa let's hold on on this, I think you want the same priority asthe other renderer which can be 0 if line width is zero. No? I'm pushing a PR with slowdown test in the second. Let's talk again after this ok?

**Comment by durack1** [comment]
@danlipsa @doutriaux1 when there is a nightly for this I would be happy to test it with the code in UV-CDAT/uvcdat#1424, that code seemed to uncover ALOT of these issues pretty well

File: vcs/vcsvtk/pipeline1d.py (hunk 1 above)
Comment (aashish24) [inline vcs/vcsvtk/pipeline1d.py]: @danlipsa @doutriaux1, why don't we remove the renderer at layer 0?

**Comment by danlipsa** [comment]
This is set in https://github.com/UV-CDAT/vcs/pull/200

File: vcs/vcsvtk/pipeline1d.py (hunk 1 above)
Comment (danlipsa) [inline vcs/vcsvtk/pipeline1d.py]: @aashish24 Good question. 😄 I don't fully understand the logic there. I think a good time to revisit this is when we implement the multiplot feature.

**Comment by doutriaux1** [comment]
@aashish24 from looking at the clear function, the layer 0 is the background color (i.e  white by default) so when we clear we repaint the background.

**Pull request closed without merging**